src/Files.Core.Storage/IFtpStorageService.cs

```diff
@@ -6,7 +6,7 @@ namespace Files.Core.Storage
 	/// <summary>
 	/// Provides an abstract layer for accessing an ftp file system
 	/// </summary>
-	public interface IFtpStorageService : IStorageService
+	public interface ISftpStorageService : IStorageService
```

Comment:
Should Sftp and Ftp be kept seperate?

Reply:
Yup that's intended but I was copy pasting certain things to save time and I accidentally changed it here. Will change 👍